### DISCUSSION TITLE
请问可以使用paddle.jit.load加载导出后的inference_model再使用train.py脚本进行训练微调吗？

### DISCUSSION THREAD

**Discussion by ad5014858** [PaddlePaddle/PaddleOCR#13388] [Q&A]
如题，流程如下：
1. 修改了yml文件发现不能识别pdiparams等后缀的文件。准备自己进行粗略代码编写。
2. 粗略的代码编写运行报错，发现应该是Dataset这块出了问题，查询了一下源代码中关于Dataloader和Dataset的代码，并没有找到DataSet的定义，故作罢。
问题：请问如何使用paddle.jit.load加载导出后的inference_model再运用train.py脚本进行训练微调？

**Comment by GreatV** [comment]
inference_model导出的是静态图。飞桨框架 2.0 及以上版本默认的编程模式是动态图模式，包括使用高层 API 编程和基础的 API 编程。如果想切换到静态图模式编程，可以在程序的开始执行 enable_static() 函数。如果程序已经使用动态图的模式编写了，想转成静态图模式训练或者保存模型用于部署，可以使用装饰器 @to_static。
https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/jit/basic_usage_cn.html

**Reply by ad5014858** [reply]
感谢解答。
